core/src/test/java/com/linecorp/armeria/common/stream/AbstractStreamMessageTest.java

```diff
@@ -17,6 +17,8 @@
 package com.linecorp.armeria.common.stream;
```

Comment:
I'm selfishly hoping #1806 gets merged first XD

Reply:
No problem at all. :-) I will remove some duplicated code after #1806 is merged.